src/main/resources/webroot/custom-js/index.js

```diff
@@ -119,19 +130,20 @@ function displayPastSessionData(data) {
     $("#past-session-data").html(compiledHtml);
 }
 
-function createCookie(value, days) {
+function createCookie(name ,value, days) {
```

Comment:
Formatting issue - please change placement of space to `createCookie(name, value, days)`